content/docs/2.10/operate/validating-webhook.md

```diff
@@ -0,0 +1,16 @@
++++
+title = "Validating Webhook"
+description = "Details on KEDA Validating Webhook"
+weight = 100
++++
+
+## Validating Webhooks
```

Comment:
```suggestion
## Admission Controller

> 💡 The Admission Controller is an opt-in feature and will become an opt-out feature as of KEDA v2.12.

```

Reply:
Is an optional feature? Using helm we can make it optional, but with `make deploy` or using kubectl apply we can't do it because the manifests are the same. That's why I configure it to not fail in case of missing admission-controller, end users can't disable them just removing or scaling to zero the deployment

Reply:
That's what we discussed on the issue; it should be an opt-in. Setting the validation to skip aligns with this here an we should make that clear or end-users will assume the validation blocks things

Reply:
Update the installation steps to reflect both files :)